npm/react/examples/react-scripts/package.json

```diff
@@ -17,6 +17,7 @@
     "react-dom": "17.0.1",
     "react-scripts": "4.0.2"
   },
+  "homepage": "/fake",
```

Comment:
What is that for?

Reply:
Regression test for this issue: https://github.com/cypress-io/cypress/issues/15811

In CRA, homepage is proxied and used for the publicPath. By adding the property here, we make sure we not have the same problem again as #15811.

Reply:
Can we write a proper test for this, rather than having an undocumented field in our `package.json`?

Reply:
Sure, how about this: https://github.com/cypress-io/cypress/pull/15839/files#diff-e426587c8b5c59a7cd0026032a63a357fdb175def624da7a340b675674776a7a

I also change fake to be more clear - it links to the issue it solves. I think we should have both, unit tests are good, but this also ensures the regression will not happen on an e2e level.